resources/roca_main.brs

```diff
@@ -0,0 +1,41 @@
+sub main()
+    basePath = "tests"
+    files = MatchFiles("pkg:/" + basePath, "*.test.brs")
+    print "# found " files.count() " files"
+    roca = "pkg:/tests/roca.brs"
+
+    rootSuites = []
+    for each file in files
+        path = ["pkg:", basePath, file].join("/")
+        suite = _brs_.runInScope(path, {})
```

Comment:
Do we expect to have all roca resources in scope at this point? My test complained about not having `describe` function defined. 
What is the  second arg in `runInScope`? I got rid of some errors by removing it

Reply:
They should, yes!  The second argument is the one (and only) argument passed into `main` function in the file defined by the first argument.  It's used here to indicate that there's no need to execute the tests, but later it contains `exec: true`!

Reply:
Oh I see, I didn't put an argument to my `main` function, that may be the issue.